LEAF_Request_Portal/Email.php

```diff
@@ -490,10 +490,11 @@ function setTemplateByLabel($emailTemplateLabel)
     /**
      * Given email template location where email addresses are stored
      * Get the email addresses, line by line, and add them if valid to CC or BCC
-     * @param $tplLocation
+     * @param string $tplLocation
+     * @param bool $isCc
      * @param false $isBcc
      */
-    function setEmailToCcWithTemplate($tplLocation, $isCc = false)
+    function setEmailToCcWithTemplate(string $tplLocation, bool $isCc = false): void
```

Comment:
Does not appear as though $isBcc is used anywhere. It is not defined within the method definition and I didn't see it being used locally. probably shouldn't be in the PHPDoc either